packages/dapp/src/helpers/parameterizer.ts

```diff
@@ -0,0 +1,28 @@
+import { Dispatch } from "redux";
+import { setParameter } from "../actionCreators/parameterizer";
+import { getParameterValue } from "../apis/civilTCR";
+
+export async function initializeParameterizer(dispatch: Dispatch): Promise<void> {
```

Comment:
how are we keeping the parameter page in sync? is there an event we can subscribe to